MarkBind/markbind
# Add Algolia plugin

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] New feature

Resolves #656.

Replaces #691.

Related PRs: #700, https://github.com/MarkBind/vue-strap/pull/98, https://github.com/se-edu/se-book/pull/66

**What is the rationale for this request?**
Allow users to integrate Agolia search with their site

**What changes did you make? (Give an overview)**
- Plugin for Agolia Search
- Update related documentation

**Testing instructions:**
For the `se-book` repository (or see https://github.com/se-edu/se-book/pull/66):

Update `site.json`:
```json
{
   ...
  "plugins": [
    "agolia"
  ],
  "pluginsContext": {
    "agolia": {
      "apiKey": "8310bb56d1ece8950bd20b0b8c080e10",
      "indexName": "se-edu"
    }
  }
  ...
}
```

Update `navigation.md`:
```md
...
      <searchbar placeholder="Search" agolia menu-align-right></searchbar>
...
```

Agolia search results should be displayed when user enters query in search bar.
